bin/migrate-oats-data/noi/sql/notice_of_intent_submission/applicant/notice_of_intent_applicant.sql

```diff
@@ -0,0 +1,43 @@
+WITH ranked_parcels AS (
+    SELECT *,
+        ROW_NUMBER() OVER (
+            PARTITION BY notice_of_intent_submission_uuid
+            ORDER BY audit_created_at
+        ) AS rn
+    FROM alcs.notice_of_intent_parcel
+),
+first_parcel_per_submission AS (
+    SELECT uuid,
+        audit_created_at,
+        notice_of_intent_submission_uuid
+    FROM ranked_parcels
+    WHERE rn = 1
+),
+ranked_owners AS (
+    SELECT *,
+        ROW_NUMBER() OVER (
+            PARTITION BY noiponoio.notice_of_intent_parcel_uuid
```

Comment:
no feedback on this, just wanted to laugh at the alias length 😄

Reply:
yeah, DBeaver likes to introduce wordy aliases